src/app/page.tsx

```diff
@@ -5,18 +5,17 @@ import styles from "./page.module.css";
 import InputText from "~/ui/InputText";
 import BtnAdd from "~/ui/BtnAdd";
 
-
 export default function Home() {
   return (
-    <div>
+    <div className="todo-list">
       <h1>To Do List</h1>
-      <div>
+      <div className="todo-list-inputText">
```

Comment:
please create a proper `page.module.css` and import these styles accordingly

Reply:
I have the styles in global.css.  Should I delete everything that contains page.module.css? and update with todo-list-inputText?

Reply:
ok!